File: Umbrella/src/main/java/com/nerdery/umbrella/data/model/TempUnit.java
package com.nerdery.umbrella.data.model;

import com.google.gson.annotations.SerializedName;

public enum TempUnit {
    @SerializedName("si")
    CELSIUS("si"),

    @SerializedName("us")
    FAHRENHEIT("us");

    private String value;

    TempUnit(String value) {
        this.value = value;
    }

    @Override
    public String toString() {
        return value;
    }
}

File: Umbrella/src/main/java/com/nerdery/umbrella/data/model/TempUnit.kt
package com.nerdery.umbrella.data.model

import com.google.gson.annotations.SerializedName

/**
 * Temperature unit to be used in requests for [com.nerdery.umbrella.data.api.WeatherService]
 */
enum class TempUnit(private val value: String) {
    @SerializedName("si")
    CELSIUS("si"),

    @SerializedName("us")
    FAHRENHEIT("us");

    override fun toString(): String {
        return value
    }
}
